resources/views/errors/generic.blade.php

```diff
@@ -1 +1,7 @@
-{{ $status_code }} {{ $status_message }}
+@extends('layouts.base')
+
+@section('content')
+    <h1>{{ $status_code }}</h1>
```

Comment:
@cydrobolt Made the changes, now we handle json request separate from web requests.
Let me know if there are any other concerns you have to merge this pull request.